mmv1/templates/terraform/pre_create/public_ca_ext_account_key.go.erb

```diff
@@ -0,0 +1,15 @@
+server_type, ok := d.GetOk("server_type")
```

Comment:
We actually generate code to override endpoints already, we shouldn't support an additional way of configuring that: https://github.com/modular-magician/terraform-provider-google-beta/compare/modular-magician:39b6b45...modular-magician:7e30707#diff-63ffdbb93783eaa496f178884f6103b3b8f97b571bcaa79b4389d74e3e51483aR447

Let's remove the `server_type` field altogether.